In [2]:
import pandas as pd

In [3]:
file_url = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vQaUVrlyhW3Mv2rayuu6rq4MnZTzIGkyUh0m3RKwAAFGtoa_AUljnZ30K_Urnjmebhtnmw1Q04O0l1j/pub?output=csv'
df = pd.read_csv(file_url)
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 918 entries, 0 to 917
Data columns (total 12 columns):
 #   Column          Non-Null Count  Dtype  
---  ------          --------------  -----  
 0   Age             918 non-null    int64  
 1   Sex             918 non-null    object 
 2   ChestPainType   918 non-null    object 
 3   RestingBP       918 non-null    int64  
 4   Cholesterol     918 non-null    int64  
 5   FastingBS       918 non-null    int64  
 6   RestingECG      918 non-null    object 
 7   MaxHR           918 non-null    int64  
 8   ExerciseAngina  918 non-null    object 
 9   Oldpeak         918 non-null    float64
 10  ST_Slope        918 non-null    object 
 11  HeartDisease    918 non-null    int64  
dtypes: float64(1), int64(6), object(5)
memory usage: 86.2+ KB


1) Source of data<br>
[Data source](https://www.kaggle.com/datasets/fedesoriano/heart-failure-prediction)<br>
2) Brief description of data<br>
Clinical features for predicting heart disease events.<br>
3) What is the target?<br>
Whether or not a patient potentially has a heart diseas<br>
4) Is this a classification or regression problem?<br>
classification<br>
5) How many features?<br>
11<br>
6) How many rows of data.<br>
918<br>
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?<br>
Cleaning up data in the object columns.